CODEOWNERS

```
@@ -1,6 +1,6 @@
 # This file is described here: https://help.github.com/en/articles/about-code-owners
 
-* @alexmt @devholic @gdsoumya @jessesuen @kencochrane @krancour @rbreeze
+* @akuity/maintainers
 
 # Doc owners
-/docs/ @dhpup @morey-tech
+/docs/ @akuity/doc-maintainers
```

```suggestion
/docs/ @akuity/doc-maintainers @akuity/doc-maintainers
```

If you do this, you will also end up closing #959

---

make sense. Accepted